hail/src/main/scala/is/hail/methods/FilterIntervals.scala

```diff
@@ -41,6 +41,13 @@ case class MatrixFilterIntervals(
   override def lower(): Option[TableToTableFunction] = Some(TableFilterIntervals(keyType, intervals, keep))
 
   def execute(mv: MatrixValue): MatrixValue = throw new UnsupportedOperationException
+
+  override def requestType(requestedType: MatrixType, childBaseType: MatrixType): MatrixType = {
```

Comment:
These will be backed out in a subsequent PR to create full IR nodes for FilterIntervals nodes.